src/Tribe/Event_Status/Event_Status_Provider.php

```diff
@@ -0,0 +1,205 @@
+<?php
+/**
+ * The Event Status service provider.
+ *
+ * @package Tribe\Events\Event_Status
+ * @since   TBD
+ */
+
+namespace Tribe\Events\Event_Status;
+
+use Tribe__Events__Main as Events_Plugin;
+use Tribe__Context as Context;
+use WP_Post;
+
+/**
+ * Class Event_Status_Provider
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Event_Status
+ */
+class Event_Status_Provider extends \tad_DI52_ServiceProvider {
+	const DISABLED = 'TEC_EVENT_STATUS_DISABLED';
```

Comment:
Docblock, something like 
```suggestion
	/**
	 * The constant used to disable event status.
	 */
	const DISABLED = 'TEC_EVENT_STATUS_DISABLED';
```